Algorithm.CSharp/RenkoConsolidatorAlgorithm.cs

```diff
@@ -29,38 +29,38 @@ public class RenkoConsolidatorAlgorithm : QCAlgorithm
         /// </summary>
         public override void Initialize()
         {
-            SetStartDate(2012, 01, 01);
-            SetEndDate(2013, 01, 01);
+            SetStartDate(2014, 05, 01);
+            SetEndDate(2014, 05, 15);
 
```

Comment:
Is there a particular reason to update RenkoConsolidatorAlgorithm? If not, we can undo the change to this file